mkosi/distributions/ubuntu.py

```diff
@@ -25,11 +25,8 @@ def default_tools_tree_distribution(cls) -> Distribution:
     def repositories(cls, context: Context, local: bool = True) -> Iterable[AptRepository]:
         types = ("deb", "deb-src")
 
-        # From kinetic onwards, the usr-is-merged package is available in universe and is required by
-        # mkosi to set up a proper usr-merged system so we add the universe repository unconditionally.
         components = (
             "main",
-            *(["universe"] if context.config.release not in ("focal", "jammy") else []),
```

Comment:
Why is the `jammy` thing being dropped here, too?

Reply:
Not required anymore, `usr-is-merged` was moved to the `main` repository